### ISSUE TITLE
CMap not displaying all edges for model

### ISSUE THREAD

**Issue by GoogleCodeExporter** [RuleWorld/rulebender#127]
```
From Leonard: "...there should definitely be edges between the egf(r) and both 
egfr_2(l) and egfr_3(l). I'm not sure why these aren't being detected, this has 
to be a bug. Does the code only consider the first action in a rule?"



```

Original issue reported on code.google.com by `AdamMatthewSmith` on 17 Feb 2013 at 11:24

Attachments:
- [egfr_net_red.bngl](https://storage.googleapis.com/google-code-attachments/rulebender/issue-127/comment-0/egfr_net_red.bngl)
